viewing_party/party.py

```diff
@@ -1,23 +1,173 @@
 # ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+
+    movie ={}
+    movie["title"] = title
+    movie["genre"] = genre
+    movie["rating"] = rating
+
+    return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for item in user_data["watchlist"]:
+        if item["title"] == title:
+            user_data["watchlist"].remove(item)
+            user_data["watched"].append(item)
+    return user_data

```

Comment:
In the future, try to avoid modifying a list as you're looping over it. You still pass the tests here, but it can lead to some strange bugs! It would be best to modify the list after the loop is finished.